### DISCUSSION TITLE
Question about typescript

### DISCUSSION THREAD

**Discussion by seloner** [TanStack/query#1823] [Q&A]
```jsx 
function useCategoriesByLevel({ level }: { level: number }) {
	return useQuery<Category[], Error>(['categories', level], () =>
		fetchCategories(level),
	);
}
```
I am using the example from the docs and and I am getting a warning missing return type on function

**Comment by TkDodo** [comment]
what warning would that be please? Could maybe be a lint rule that you have enabled...
